src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201512090753.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ContentBundle;
+
+use PHPCR\Migrations\VersionInterface;
+use PHPCR\NodeInterface;
+use PHPCR\SessionInterface;
+
+class Version201512090753 implements VersionInterface
```

Comment:
Please add a describing header, since the class name does not suggest what this class is actually doing.